scss/inc/_base.scss

```diff
@@ -396,6 +396,10 @@ body {
         margin: 20px 0 20px 20px;
     }
 
+    .center {
```

Comment:
Let's rename it something more specific, `tao-centered` or `tao-center-image`? I know `wrap-left/wrap-right` are generic names, but maybe it's safer not to conflict with anything (in custom item styles too).